### ISSUE TITLE
🐛 Bug: Ersilia Setup problem

### ISSUE THREAD

**Issue by Dorothy2020** [ersilia-os/ersilia#334]
### Describe the bug.

Ersilia serve models cannot be fetched

### Describe the steps to reproduce the behavior

https://www.oreilly.com/library/view/ssh-the-secure/0596008953/ch07s03.html
Click on the link to solve the error in verbose mode

### Expected behavior.

EmptyOutputError

### Screenshots.

![Screenshot from 2022-10-14 07-39-43](https://user-images.githubusercontent.com/79142184/195764601-f59b3d08-6778-4466-9adb-fec46b5270c4.png)

### Operating environment

Ubuntu 20.04

### Additional context

https://www.oreilly.com/library/view/ssh-the-secure/0596008953/ch07s03.html

**Comment by Femme-js** [comment]
HI @Dorothy2020 

Have you fetched the model first?

Usually, You create the conda environment first, install all the requirements, and then fetch the model. After fetching you can serve the models. 

From the error I can see, you might not have installed it. (pip install -e .)

**Comment by Dorothy2020** [comment]
Hello @Femme-js , yes I did fetch the model and it worked perfectly , so when I write ersilia serve command that's when I get the error

**Comment by GemmaTuron** [comment]
Hi @Dorothy2020 

We need to see the log of the error in order to provide guidance. Please add the -v flag and save the error log.
Also, I see you jumped directly to testing this model but I haven't seen you comment on the first steps, which are testing that the CLI works and testing a simple model, eos3b5e. 
Please follow the steps as some models are quite complex and before using them you must make sure ersilia is properly installed

**Comment by GemmaTuron** [comment]
Hi @Dorothy2020 

I am just following on the thread, can you confirm if ersilia works? Try:
`conda activate ersilia`
`ersilia --help`
If this works fine, then try to fetch the model and store the log file.

**Comment by Dorothy2020** [comment]
@GemmaTuron I was able to install the ersilia successfully following ersilia-book: https://ersilia.gitbook.io/ersilia-book:

1.Activated ersilia using conda activate ersilia.
2. Checked on the help using command ersilia --help
3. Check on the log using command ersilia catalog
![Screenshot from 2022-10-23 15-31-53](https://user-images.githubusercontent.com/79142184/197453090-f127fff1-8f25-4e38-b0a0-b675815a428e.png)

**Screenshoot for the sucessfull installation of ersilia:**


**Tested few models using the following commands:**
1. Activated ersilia using conda activate ersilia
2. Fetched the models using  ersilia -v fetch model eos6oli
3. Serve the model using command ersilia server eos6oli
4. Run the predictions using command ersilia api -i 'CCCOCCC'


**Operating system:**
Ubuntu 20.04.5
Python version: Python 3.7.13
conda version :conda 22.9.0
